app/controllers/api.js

```diff
@@ -8,7 +8,7 @@ API.checkOut = function (req, res) {
 	// TODO
     //Get string data from card and send to other API - get studentID back
     var cardString = req.body.cardString;
-    var dockID = req.body.dockID;
+    var dockID = req.body.macAddress;
```

Comment:
The mac will still be transferred as "dockID" in the json, but will come across as a string instead of a number as before.